WalletWasabi/Wallets/Wallet.cs

```diff
@@ -349,7 +355,7 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
 		{
 			await PerformSynchronizationAsync(SyncType.NonTurbo, stoppingToken).ConfigureAwait(false);
 		}
-		Logger.LogInfo($"Wallet '{WalletName}' is fully synchronized.");
+		Logger.LogInfo($"Wallet {this} is fully synchronized.");
```

Comment:
It shows the Id and the name, since wallets are now internally identified by their Id, for consistency.

Reply:
ok, I'm asking because then we now have two different ways of logging the wallet name.

---

732e1a47b5feb18ffee69a19f5d49e7fb5b966d7 fixed Ctrl log issue https://github.com/zkSNACKs/WalletWasabi/pull/11652#pullrequestreview-1724479451

Reply:
Unfortunately, some of our core still use names as the unique identifier of the wallets, so we can't do anything for now.